app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -51,71 +51,23 @@
     }
   }
 
-  .contents-list-wrapper {
+  .column-quarter-desktop-only {
     @include grid-column( 1 / 4, $full-width: desktop );
 
     .direction-rtl & {
       float: right;
-      direction: rtl;
-      text-align: start;
-    }
-
-    .contents-list {
-      position: relative;
-      z-index: 1;
-      padding-bottom: $gutter;
-      list-style: none;
-      background: $white;
-      box-shadow: 0 20px 15px -10px $white;
-
-      li {
-        padding-top: $gutter / 4;
-        line-height: 1.3;
-      }
-
-      .contents-text,
-      .contents-number {
-        display: table-cell;
-      }
-
-      .contents-number {
-        min-width: 1.5em;
-      }
-
-      .contents-text {
-        $contents-text-padding: 0.3em;
-        padding-left: $contents-text-padding;
-
-        .direction-rtl & {
-          padding-left: 0;
-          padding-right: $contents-text-padding;
-        }
-      }
-
-      a {
-        display: table;
-        text-decoration: none;
-
-        &:hover {
-          text-decoration: underline;
-        }
-      }
-
-      @include media(false, $desktop-breakpoint) {
-        box-shadow: none;
-      }
     }
   }
 
-  .contents-wrapper {
+  .column-three-quarters-desktop-only {
     @include grid-column( 3 / 4, $full-width: desktop );
```

Comment:
This should be the same as before (ie as added in #384), classes are just renamed.
Did you compare with live?